src/customHooks/useLeaderBoard/useLeaderBoard.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from "@tanstack/react-query";
+import { getLeaderBoardFromLocalStorage } from "../../api/localStorageApi";
+import { errorMessage } from "./strings";
+
+
+export function useLeaderBoard(gameId: number) { 
+    const { data } = useQuery([gameId.toString], getLeaderBoardFromLocalStorage, {
+        onError: (err) => errorMessage + err,

```

Comment:
You need to throw an error for this to work

Reply:
you can test that this works properly by throwinng an error at getLeaderBoardFromLocalStorage